dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -1914,8 +1915,22 @@ public Entry registerCheckerWorkflow(@ApiParam(hidden = true) @Parameter(hidden
      * @param workflow
      */
     private void initializeAdditionalFields(String include, Workflow workflow) {
+        initializeAdditionalFields(include, workflow, true);
+    }
+
+    /**
+     * If include contains validations field, initialize the workflows validations for all of its workflow versions If include contains aliases field, initialize the aliases for all of its workflow
```

Comment:
The  comment formatting looks a bit weird. Either missing periods or line breaks or both. 
Would assume from the structure that each "If include" section should be on a new line.